paasta_tools/paasta_metastatus.py

```diff
@@ -286,6 +286,26 @@ def print_results_for_healthchecks(summary, ok, results, verbose):
             print_with_indent(line, 2)
 
 
+def print_extra_mesos_data():
```

Comment:
Add at least one test for this, and make the function just return a string and let the caller be in charge of printing.

Input an example mesos state and assert that some of the expected strings are there.

This state file is going to change as we upgrade mesos, it will be easier to be sure what breaks if we can inject it at the unit test level